src/Php72/Php72.php

```diff
@@ -95,6 +95,35 @@ public static function spl_object_id($object)
         return self::$hashMask ^ hexdec(substr($hash, 16 - \PHP_INT_SIZE, \PHP_INT_SIZE));
     }
 
+    public static function sapi_windows_vt100_support($stream, $enable = null)
+    {
+        // We cannot actually enable vt100 support
+        if (true === $enable || !self::stream_isatty($stream)) {
+            return false;
```

Comment:
I'm just wondering if I have got the logic right here, in automatically returning false if the enable param is true. It makes sense in the strictest way, in that you cannot actually enable the setting without using the native function. But in terms of how the polyfill will used, it may be better to remove this bit (because an application would probably always set the enable param to true). Thoughts please.

Also, I haven't covered the case where enable = false.

Reply:
Perhaps `if (false === $enable ...` would be better, in that one would simply not apply the vt100 formatting sequences, regardless of what the native function would return.